src/screens/HowItWorks.tsx

```diff
@@ -0,0 +1,114 @@
+import { Alert, Linking, StyleSheet, View, FlatList } from "react-native";
+import React from "react";
+import CarbonText from "../components/CarbonText";
+import { FontSizeMedium } from "../FontSizes";
+import CarbonButton from "../components/CarbonButton";
+import CarbonButtonOutline from "../components/CarbonButtonOutline";
+import { Gray100, White0 } from "../CarbonColors";
+import SVGIcon from "../components/SVGIcon"
+import { useHistory } from "react-router-dom";
+
+function HowItWorks() {
+
+    const history = useHistory();
+
+      function userDetailsOnClick() {
+        history.push("/userDetails");
+      }
+
+      function openURL(url: string) {
+        Linking.canOpenURL(url).then((supported) => {
+          if (supported) return Linking.openURL(url);
+          Alert.alert("Error", "Could not open URL", [{ text: "Ok" }]);
+        });
+      }
+
+    const howItWorksSteps = [
+        {
+            step: '1',
+            icon: 'iconUSBC',
+            title: 'Connect sensor to USB-C'
+        },
+        {
+            step: '2',
+            icon: 'iconEdit',
+            title: 'Enter your info in the app'
+        },
+        {
+            step: '3',
+            icon: 'iconWifi',
+            title: 'Enable internet'
+        },
+        {
+            step: '4',
+            icon: 'iconIot',
+            title: 'Add sensor to network'
+        },
+        {
+            step: '5',
+            icon: 'iconLaptop',
+            title: 'View your data on the dashboard'
+        },
+    ];
+
+    const Step = ({ title, icon }) => (
```

Comment:
Please use interfaces.

Ex:
```
interface StepProps {
    title: string;
    icon: string; // if it actually is a string
}
```

The interface can then be used to define the function the following way:
`function Step(props: StepProps) {`